PaddleNLP/language_representations_kit/BERT/optimization.py

```diff
@@ -19,7 +19,7 @@
 
 import numpy as np
 import paddle.fluid as fluid
-from utils.fp16 import create_master_params_grads, master_param_to_train_param
+from utils.fp16 import create_master_params_grads, master_param_to_train_param, apply_dynamic_loss_scaling
```

Comment:
Use decorate.

Reply:
The decorate interface should be refined, before that we use the customized code here.